WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -134,7 +132,6 @@ public WordPressDB(Context ctx) {
                 ctx.deleteDatabase("simperium-store");
                 currentVersion++;
             case 37:
-                ThemeTable.resetTables(db);
```

Comment:
Shouldn't we delete the tables for users that are upgrading from after version 37?